movies/src/main/java/comp.fundacionjala/movies/IMovie.java

```diff
@@ -0,0 +1,8 @@
+package comp.fundacionjala.movies;
+
+public interface IMovie {
+
+    double calculateRentalCharge(int daysRented);
+
+    int calculateRenterFrequentPoints(int daysRented);
```

Comment:
@carledriss 

> This implementation should be implemented in the NewRelease, Children, and Regular class
Update your code

to implement this method in the mentioned classess(ChildrenMovie, RegularMovie, and NewReleaseMovie) according to the last code review observation,  i used the IMovie interface, i think that is incorrect way to implement this method but i cannot found a better way.